tests/Functional/Entity/TaskExecutionRepositoryTest.php

```diff
@@ -0,0 +1,215 @@
+<?php
+
+namespace Task\TaskBundle\Tests\Functional\Command;
+
+use Task\Execution\TaskExecutionInterface;
+use Task\Storage\TaskExecutionRepositoryInterface;
+use Task\Storage\TaskRepositoryInterface;
+use Task\TaskBundle\Entity\Task;
+use Task\TaskBundle\Entity\TaskExecution;
+use Task\TaskBundle\Tests\Functional\BaseDatabaseTestCase;
+use Task\TaskBundle\Tests\Functional\TestHandler;
+use Task\TaskInterface;
+use Task\TaskStatus;
+
+/**
+ * Tests for TaskExecutionRepository.
+ */
+class TaskExecutionRepositoryTest extends BaseDatabaseTestCase
+{
+    /**
+     * @var TaskExecutionRepositoryInterface
+     */
+    protected $taskExecutionRepository;
+
+    /**
+     * @var TaskRepositoryInterface
+     */
+    protected $taskRepository;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->taskExecutionRepository = self::$kernel->getContainer()->get('task.storage.task_execution');
+        $this->taskRepository = self::$kernel->getContainer()->get('task.storage.task');
+    }
+
+    public function testSave(TaskInterface $task = null, \DateTime $scheduleTime = null, $status = TaskStatus::PLANNED)
```

Comment:
I would exclude it in another function... There is no need that these tests are executed multiple times within a single run.